app/src/main/res/values/strings.xml

```diff
@@ -82,11 +84,16 @@
         <item quantity="one">%d day</item>
         <item quantity="other">%d days</item>
     </plurals>
+    <string name="textFieldOptional">(Optional)</string>
+    <string name="transferMessagePlaceholder">A message to pass on?</string>
+    <string name="transferRecipientAddressPlaceholder">Send the files to</string>
     <string name="transferSendButton">Send</string>
+    <string name="transferSenderAddressPlaceholder">Your email address</string>
     <plurals name="transferSpaceLeft">
         <item quantity="one">%s left</item>
         <item quantity="other">%s left</item>
     </plurals>
+    <string name="transferTitlePlaceholder">Title your transfer</string>
```

Comment:
I don't know what this was supposed to mean but the wording feels strange, does it not?

Reply:
Well it wasn't from me and I understood it instinctly, so I guess it doesn't feel that strange
It's the string for the transfer title field's placeholder

Reply:
Turns out this wording shouldn't even exists because the Figma was wrong, the API doesn't allow this field.
So I removed this

Reply:
Wouldn't something like "Give a title to your transfer" feel more natural?

Reply:
We don't really care anymore, as I say this string shouldn't have existed in the first place. This feature don't exists

Reply:
Oh my bad, I didn't get what you meant, no problem then!